### DISCUSSION TITLE
How to model blast loading simulation using ls dyna which can be solved in OpenRadioss?

### DISCUSSION THREAD

**Discussion by JaswanthGangolu** [OpenRadioss/OpenRadioss#3112] [Users]
Hi, 

Not all the keywords in ls dyna are supported by the solver of OpenRadioss. What is keyword suppose to be used for blast loading? Does anyone share some example file of blast loading? 

Thank you.

**Comment by RevolT87** [comment]
Hi @JaswanthGangolu,
Currently we are not supporting LS-Dyna keywords for blast or ALE framework. What is the LS-Dyna keyword you want to use? 
Is working with Radioss format a blocking point for you? 
In Radioss we have /LOAD/PBLAST to represent directly the impulse coming from a blast on the structure. 
I hope this helps.
Regards,
Mathis

**Comment by JaswanthGangolu** [comment]
I am new to OpenRadioss and need some guidance to get started. Which
software should I use to create a model file that I can run in OpenRadioss?
*Dr. Jaswanth Gangolu*
*Technion Post Doctoral Researcher*
*Extreme Loads on Structures Lab*

*Technion Israel Institute of Technology*
*Contact: +91-7569265953, +972-555562544 *
*Google Scholar
<https://scholar.google.com/citations?user=tycibrEAAAAJ&hl=en>*   *Research
Gate <https://www.researchgate.net/profile/Jaswanth-Gangolu>*   *Linkedin
<https://www.linkedin.com/public-profile/settings?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_self_edit_contact-info%3BI9Pi9yIjTnmXLmgNdU3ePA%3D%3D>*
*Website <https://elsl.technion.ac.il/group-members/>*

**Comment by JaswanthGangolu** [comment]
I am new to OpenRadioss and need some guidance to get started. Which software should I use to create a model file that I can run in OpenRadioss?

**Reply by MarianBulla** [reply]
Hi @JaswanthGangolu,
thanks for your interest in OpenRadioss. As mentioned in the documenation page of OpenRadioss (on the website: www.openradioss.org ) you can use several open-source and industrial pre- and post-processors for your work. 
As student or an employee of an university, you can have also access to the student edition of HyperMesh, which is one of the best tool to get started. (also mentioned on the docu page of OpenRadioss).
There are also lot of examples, where you can have a look.

Hope it helps you ?

Thanks and best regards,
Marian
